### PR TITLE
Update build instructions

### PR DESCRIPTION
**Pull request by RussKie**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/3746)

### CODE REVIEW

File: docs/building.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+1. Right click on the project you wish to run and select 'Set as startup project'.
+2. <kbd>F5</kbd>.
+
+## Running apps from command line
+
+1. Build the solution: `.\build.cmd`
+2. Navigate to the project you wish to run, e.g. to run AccessibilityTests test app:
+    ```
+    winforms> pushd .\src\System.Windows.Forms\tests\AccessibilityTests>
```
Comment (Olina-Zhang) [inline docs/building.md:46]: Remove right angle bracket, it should be **winforms> pushd .\src\System.Windows.Forms\tests\AccessibilityTests**

**Review by Olina-Zhang — CHANGES_REQUESTED** [review]

File: docs/building.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+3. (Alternatively) Navigate to .\artifacts\bin\AccessibilityTests\Debug\netcoreapp5.0 and run the app manually.
+
+## Troubleshooting build errors
+
+* Most build errors are compile errors and can be dealt with accordingly.
+* Other error may be from MSBuild tasks. You need to examine the build logs to investigate.
+  * The logs are generated at `artifacts\log\Debug\Build.binlog`
+  * The file format is an MSBuild Binary Log. Install the [MSBuild Structured Log Viewer][msbuild-log-viewer] to view them.
+* WinForms uses Visual Studio MSBuild but sometimes with a preview .NET Core SDK; so if you have a non-preview version of [Visual Studio][VS-download] (i.e. a release version), then you may need to enable `use preview` for .NET Core SDKs in VS.
+  * you can do this in VS at Tools :arrow_right: options :arrow_right: Projects and Solutions :arrow_right: .net core :arrow_right: use previews
```
Comment (Olina-Zhang) [inline docs/building.md:58]: This option is moved to Tools->Options->Environment->Preview Features->Use previews of the .Net Core SDK (Requires restart)

File: docs/building.md [hunk 3]
```diff
@@ -4,35 +4,49 @@
...
+1. Run `.\start-vs.cmd` from the repository root. This will prepend the repo-local .NET SDK to the path, and open `Winforms.sln` in Visual Studio.
+1. You should now be able to build as you normally would.
 
-* To build from Visual Studio, open `Winforms.sln` in Visual Studio and build how you normally would.
-* Visual Studio behaves slightly differently than the command line. It uses the machine-wide installed SDK instead of the one specified in the global.json.
-  * Please make sure you have the [latest .Net Core Daily Build][latest-core-build] installed.
+### Building from Visual Studio Code
+
+1. (Optional) Run `.\restore.cmd` from the repository root.
+1. Run `.\start-code.cmd` from the repository root. This will prepend the repo-local .NET SDK to the path, and open `Winforms.sln` in Visual Studio.
```
Comment (hughbe) [inline docs/building.md:25]: nit: numbering

Comment (hughbe) [inline docs/building.md:25]: nit: Visual Studio Code, not Visual Studio

File: docs/building.md [hunk 4]
```diff
@@ -4,35 +4,49 @@
...
+## Running apps from VS
+
+1. Right click on the project you wish to run and select 'Set as startup project'.
+2. <kbd>F5</kbd>.
+
+## Running apps from command line
+
+1. Build the solution: `.\build.cmd`
+2. Navigate to the project you wish to run, e.g. to run AccessibilityTests test app:
+    ```
```
Comment (hughbe) [inline docs/building.md:45]: nit:indent?

File: docs/building.md (hunk 3 above)
Comment (RussKie) [reply docs/building.md:25]: Thank you, but MD is smart enough to automatically increment numbering.

File: docs/building.md (hunk 4 above)
Comment (RussKie) [reply docs/building.md:45]: It is correct, I want it indented.


Unindented it would look like this:
image

**Pull request merged**